src/libs/OptionsListUtils.js

```diff
@@ -436,6 +441,12 @@ function getLastMessageTextForReport(report) {
         lastMessageTextFromReport = lodashGet(lastReportAction, 'message[0].text', '');
     } else if (ReportActionUtils.isCreatedTaskReportAction(lastReportAction)) {
         lastMessageTextFromReport = TaskUtils.getTaskCreatedMessage(lastReportAction);
+    } else if (ReportActionUtils.isMemberChangeAction(lastReportAction)) {
+        const htmlWithoutMutedTags = lodashGet(lastReportAction, 'message[0].html', '').replace(/<[/]?muted-text>/g, '');
+        const htmlWithoutAccountIDBasedMentions = ReportActionUtils.convertAccountIDBasedMentionsToDisplayNames(htmlWithoutMutedTags, allPersonalDetails);
+        lastMessageTextFromReport = htmlWithoutAccountIDBasedMentions.replace(/<a[^>]*>(.*?)<\/a>/gi, '$1');
```

Comment:
Does this support localization?

Reply:
If you're asking about the "invited" text, then no, it does not currently support localization because that is generated in the back-end.

Reply:
ok, but I am seeing that other customized system messages support localization, though not supported yet in backend. Especially in `ModifiedExpenseMessage.ts`

Reply:
Can we address this in a different PR? This one is intended to solve a very specific issue.

Reply:
It's been open for a bit too long and I'd like to get it merged soon.